lmnet/tests/lmnet_tests/networks_tests/test_quantize_param_init.py

```diff
@@ -0,0 +1,115 @@
+import tensorflow as tf
```

Comment:
I wrote this:
> I hope to implement unit tests using `QuantizeParamInit` mixin class with new network class
> 
>     * When the param `quantize_first_convolution=True` and `quantize_last_convolution=True`, the all of network's activations before convolution and weights are quantized.
> 
>     * When the param `quantize_first_convolution=True` and `quantize_last_convolution=False`, the all of network's activations before convolution and weights are quantized excluding last convolution.
> 
>     * When the param `quantize_first_convolution=False` and `quantize_last_convolution=True`, the all of network's activations before convolution and weights are quantized excluding first convolution.

Meaning of `new network class` is to implement simple network class only for test on this file to avoid dependency, to simplify :smile:

Reply:
Does this mean implementing the unit test on the `new network class` is enough?
No need to implement it with the existing network, e.g. lmnet.

Reply:
`test_required_arguments()` is not need. Because first and last layer are already asserted in `QuantizeParamInit`.

My intention was only the new test class, but I think it is good to test LMNetQuantize etc, also.
However, When I implement new quantize network, I might worry about  whether should add the new network to the test. How do you think?

Reply:
I got your point now.
I think that the test class function should check the new class too, and automatically if possible.
So I will find a way to get all quantized classes automatically from the module and check them.

However, if I cannot find the way, developers of the new class must import that new class in the test manually.
I will update later.